### PR TITLE
Revive the berts, and some ernies too

### PR DESCRIPTION
**Pull request by Licenser**
# Pull request

## Description

Updates the bert-related operators to allow usage of them again

## Checklist

* [x] The RFC, if required, has been submitted and approved
* [x] Any user-facing impact of the changes is reflected in docs.tremor.rs
* [x] The code is tested
* [x] Use of unsafe code is reasoned about in a comment
* [x] Update CHANGELOG.md appropriately, recording any changes, bug fixes, or other observable changes in behavior
* [x] The performance impact of the change is measured (see below)

## Performance
no

### CODE REVIEW

**Review by darach — APPROVED** [review]
LGTM 👍 // Sentiment: 0.99 positive

File: .vscode/settings.json [hunk 1]
```diff
@@ -128,5 +128,8 @@
     "whrf",
     "whrt",
     "wintel"
-  ]
+  ],
+  "rust-analyzer.cargo.features": [
+    "bert"
```
Comment (mfelsche) [inline .vscode/settings.json:133]: won't this disturb e.g. working on tremor-script alone?

**Review by mfelsche — APPROVED** [review]
otherwise LGTM

File: .vscode/settings.json (hunk 1 above)
Comment (Licenser) [reply .vscode/settings.json:133]: Sorry about that, that slipped through I thought vscode was in ignore!

Comment (Licenser) [reply .vscode/settings.json]: fixed

**Review by mfelsche — APPROVED** [review]

**Pull request merged**